sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -1591,12 +1597,12 @@ public void testArrayStuff() throws SQLException
         Assert.assertArrayEquals(new Object[]{"a", null, "", "a", "abc", null}, (Object[]) rows.get(0).get("arr1"));
         Assert.assertArrayEquals(new Object[]{7L, 325323L, 0L, null, null, null}, (Object[]) rows.get(0).get("arr2"));
         Assert.assertArrayEquals(new Object[]{1.0, 1.7, 0.0, null, null, null}, (Object[]) rows.get(0).get("arr3"));
-        Assert.assertArrayEquals(new Object[]{1.0f, 0.1f, 0.0f, null, null, null}, (Object[]) rows.get(0).get("arr4"));
+        Assert.assertArrayEquals(new Object[]{1.0, 0.10000000149011612, 0.0, null, null, null}, (Object[]) rows.get(0).get("arr4"));
```

Comment:
what caused the result to change?

Reply:
ARRAY_AGG on a float column returns the same values when run in master. This might be a bug on to as how ARRAY_AGG is handling return types in general